R/github-key.R

```diff
@@ -80,11 +86,14 @@ create_key_data <- function(pubkey, title) {
 #' @param info `[list]`\cr
 #'   GitHub information for the repository, by default obtained through
 #'   [github_info()].
+#' @template remote
 #'
 #' @export
 #' @keywords internal
 #' @rdname github_info
-github_repo <- function(path = usethis::proj_get(), info = github_info(path)) {
+github_repo <- function(path = usethis::proj_get(),
```

Comment:
Can we adopt the usethis model of the "current project" here (after merging this PR)?

Reply:
`package = NULL`?